### DISCUSSION TITLE
2FA not working, lost recovery codes and ssh key.

### DISCUSSION THREAD

**Discussion by pratikmm** [community/community#31753] [New to GitHub] [answered]
Hi Lovely people,
I need help to recover my github account access. I know my username and password but my 2FA code is not working. Tried restting it but it asks for recovery codes or SSH keys, both of which I don't have. Right now I just have access to my email account associated with github user and its username and password. Has anyone faced such situation? Please help me to recover the access to my account.
Thanks in advance,

Regards,
Pratik

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Good luck! :)

**Reply by MichaelTuchman** [reply]
The problem with this approach is that when you go to the page, none of the categories offered (accounts/billing, etc.) relates to problems with authentication.  There needs to be a category just for authentication problems.

**Comment by pratikmm** [comment]
Hi @queenofcorgis , Thank you very much for the response. I have created the support ticket for the same few days ago. Is there way to prioritise this? https://support.github.com/ticket/personal/0/1774810
Its very urgent as its my work account and I am completely blocked to work.

**Reply by queenofcorgis** [reply]
Hi there @pratikmm - Myself and the rest of the moderator team are not able to escalate Support tickets. I can confirm that your ticket has been received and is in the proper place. Thank you for your patience.

**Reply by pratikmm** [reply]
Hi @queenofcorgis , Thank you very much for your reply. I can understand you cannot escalate my support ticket. But its been 7 days I have created the [ticket](https://support.github.com/ticket/personal/0/1774810). Is there any estimation can be given to resolve the ticket from github side my for ticket? I need this so that I can communicate the same to my organisation. Thank you.
Regards,
Pratik

**Reply by pulsenightlife** [reply]
> Hi there @pratikmm - Myself and the rest of the moderator team are not able to escalate Support tickets. I can confirm that your ticket has been received and is in the proper place. Thank you for your patience.

Hello, I am dealing with an issue like this aswell. Please help look at my ticket if you can. Thank you

**Comment by atiwari-athene** [comment]
Hi, 

My team member is not able to login using neither 2 factor authentication nor recovery code. For both cases it is showing her invalid code. Please help her to resolve this issue. Thanks in advance

**Reply by queenofcorgis** [reply]
The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Good luck! :)

**Reply by ghost** [reply]
Hi,

i face same issue here .. my MacBook has power issue and dose not turn on , so i am unable to get ssh & recovery code 
i have 2FA code "installed on mobile" and have access to my primary & secondary email .
more details added in (#1811901) https://github.com/Elshaikh ..
appreciate feedback to get back to work 

 Thanks in advance

**Reply by abdullahsohailyaqoob273322** [reply]
I am also facing the same with my old user
@abdullahsohailyaqoob
